modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -0,0 +1,147 @@
+	description = "Used to encourage recovery of internal organs and nervous systems. Medicate cautiously."
+
+/singleton/reagent/ryetalyn
+	description = "Ryetalyn can cure all genetic abnomalities via a catalytic process."
+
+/singleton/reagent/hyperzine
+	description = "Hyperzine is a highly effective, long lasting, muscle stimulant. Lasts twice as long when inhaled."
+
+/singleton/reagent/ethylredoxrazine
+	description = "A powerful oxidizer that reacts with ethanol."
+
+/singleton/reagent/hyronalin
+	description = "Hyronalin is a medicinal drug used to counter the effect of radiation poisoning."
+
+/singleton/reagent/arithrazine
+	description = "Arithrazine is an unstable medication used for the most extreme cases of radiation poisoning."
+
+/singleton/reagent/thetamycin
+	description = "A theta-lactam antibiotic, effective against wound and organ bacterial infections."
+
+/singleton/reagent/asinodryl
+	name = "Ondansetron"
+	description = "Ondansetron is a medication used to prevent nausea and vomiting."
+
+/singleton/reagent/antidexafen
+	name = "Cough Syrup"
+	description = "A chemical that is used as a cough suppressant in low doses, and in higher doses it can be recreationally (ab)used."
+
+/singleton/reagent/cetahydramine
+	name = "Diphenhydramine"
+	description = "A common antihistamine medication, also known as Benadryl. Known for causing drowsiness in larger doses."
+
+/singleton/reagent/sterilizine
+	description = "Sterilizes wounds in preparation for surgery and thoroughly removes blood."
+
+/singleton/reagent/leporazine
+	description = "Leporazine can be use to stabilize an individual's body temperature."
+
+/singleton/reagent/cataleptinol
+	name = "Mannitol"
+	description = "Mannitol is a super strength chemical that heals brain tissue damage and cures dumbness, cerebral blindness, cerebral paralysis, colorblindness, and aphasia. More effective when the patient's body temperature is less than 170K."
+
+/singleton/reagent/fluvectionem
+	name = "Calomel"
+	description = "A highly toxic medicine that quickly purges most chemicals from the bloodstream. Overdose causes bloodloss and more toxin buildup, however works twice as fast."
+
+/singleton/reagent/pulmodeiectionem
+	description = "A powdery chemical that damages the mucus lining in the the main broncus and the trachea, allowing particles to easily escape the lungs. Only works when inhaled. May cause long term damage to the lungs, and oxygen deprevation."
+
+/singleton/reagent/pneumalin
+	description = "A chemical that, when inhaled, restores tearing and bruising of the lungs. Overdosing can lower a patient's pulse to dangerous levels."
+
+/singleton/reagent/rezadone
+	description = "A powder with almost magical properties, this substance can effectively treat genetic damage in humanoids, though excessive consumption has side effects."
+
+/singleton/reagent/verunol
+	name = "Ipecac"
+	description = "A simple emetic, Induces vomiting in the patient, emptying stomach contents"
```

Comment:
"A complex emetic medication that causes the patient to vomit due to gastric irritation and the stimulation of the chemoreceptor trigger zone."